kms/kms.pb.go

```diff
@@ -27,6 +27,7 @@ const (
 	FheType_Euint8  FheType = 1
 	FheType_Euint16 FheType = 2
 	FheType_Euint32 FheType = 3
+	FheType_Euint64 FheType = 4
```

Comment:
This file is auto-generated, so you would need to update the `proto/kms.proto` file and regenerate